# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_bug482934.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_bug482934.js
--- a/netwerk/test/unit/test_bug482934.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_bug482934.js	Tue Feb 12 16:08:25 2019 +0000
@@ -92,7 +92,7 @@
   response_time = "Thu, 1 Jan 2009 00:00:00 GMT";
 
   var ch = make_channel(resource_url);
-  ch.asyncOpen2(new ChannelListener(function(aRequest, aData) {
+  ch.asyncOpen(new ChannelListener(function(aRequest, aData) {
     Assert.ok(hit_server);
     Assert.equal(aRequest.QueryInterface(Ci.nsIHttpChannel).responseStatus, 304);
     Assert.ok(!cache_storage.exists(make_uri(resource_url), ""));
@@ -111,7 +111,7 @@
   response_time = "Fri, 2 Jan 2009 00:00:00 GMT";
 
   var ch = make_channel(resource_url);
-  ch.asyncOpen2(new ChannelListener(function(aRequest, aData) {
+  ch.asyncOpen(new ChannelListener(function(aRequest, aData) {
     Assert.ok(hit_server);
     Assert.equal(aRequest.QueryInterface(Ci.nsIHttpChannel).responseStatus, 200);
     Assert.ok(cache_storage.exists(make_uri(resource_url), ""));
@@ -130,7 +130,7 @@
   response_time = "Fri, 2 Jan 2009 00:00:00 GMT";
 
   var ch = make_channel(resource_url);
-  ch.asyncOpen2(new ChannelListener(function(aRequest, aData) {
+  ch.asyncOpen(new ChannelListener(function(aRequest, aData) {
     Assert.ok(hit_server);
     Assert.equal(aRequest.QueryInterface(Ci.nsIHttpChannel).responseStatus, 304);
     Assert.ok(cache_storage.exists(make_uri(resource_url), ""));
@@ -150,7 +150,7 @@
   request_time = "Fri, 2 Jan 2009 00:00:00 GMT";
   response_time = "Sat, 3 Jan 2009 00:00:00 GMT";
   var ch = make_channel(resource_url);
-  ch.asyncOpen2(new ChannelListener(function(aRequest, aData) {
+  ch.asyncOpen(new ChannelListener(function(aRequest, aData) {
     Assert.ok(hit_server);
     Assert.equal(aRequest.QueryInterface(Ci.nsIHttpChannel).responseStatus, 200);
     Assert.ok(cache_storage.exists(make_uri(resource_url), ""));